src/blobscan/blobscan_launcher.star

```diff
@@ -184,11 +211,12 @@ def get_web_config(
         ports=WEB_PORTS,
         public_ports=public_ports,
         env_vars={
-            "DATABASE_URL": database_url,
-            "SECRET_KEY": "supersecret",
-            "NEXT_PUBLIC_NETWORK_NAME": "devnet",
-            "BEACON_NODE_ENDPOINT": beacon_node_rpc,
-            "CHAIN_ID": chain_id,
+            "DATABASE_URL": postgres_url,
+            "NEXT_PUBLIC_NETWORK_NAME": network_name
+            if network_name in constants.PUBLIC_NETWORKS
+            else "devnet",
+            "NEXT_BEACON_NODE_ENDPOINT": beacon_node_rpc,
+            "EXECUTION_NODE_ENDPOINT": execution_node_rpc,
```

Comment:
This env vars are only required by the indexer.

The web app doesn't use them.

Reply:
Removed.